examples/ising/run.py

```diff
@@ -0,0 +1,28 @@
+import solara
+from ising.model import IsingModel
+from ising.portrayal import portray_spin
+from mesa.visualization import JupyterViz
+from mesa.visualization.UserParam import Slider
+
+model_params = {
+    "temperature": Slider(
+        label="Temperature", value=2, min=0.01, max=10, step=0.06, dtype=float
+    ),
+    "spin_up_probability": Slider(
+        label="Spin Up Probability",
+        value=0.5,
+        min=0,
+        max=1,
+        step=0.05,
+    ),
+}
+
+
+@solara.component
+def Page():
+    JupyterViz(
```

Comment:
```suggestion
    SolaraViz(
```

Reply:
I dont know if I understood how to update the visualization correctly. are we using the Streamlit platform for default now?